app/views/requests/request_mailer/service_error_email.html.erb

```diff
@@ -1,5 +1,4 @@
-==============================================
-Remote Service Request Error
+<h1>Remote Service Request Error<h1>
```

Comment:
@carolyncole If add 
```
<tr>
  <td colspan="2" style="padding: 18px;">
``` 
above line 1 and 
```
  </td>
</tr>
```
after line 4, you should get the template to render the white background